crdppf/static/js/Crdppf/map.js

```diff
@@ -70,17 +70,84 @@ var setInfoControl = function setInfoControl(){
         box: false,
         hover: false,
         single: false,
-        maxFeatures: 1,
+        maxFeatures: 4,
         clickTolerance: 15
     });
-    
-    // define actions on feature selection
-    control.events.register("featureselected", this, function(e) {
+        
+    Crdppf.PropertySelection = function(features, labels) {
+        
+        if (store) {
+            store.removeAll();
+        }
+        
+        if (features.length > 1) {
+            var properties = []
+            for (var i = 0; i < features.length; i++){
+                properties[i] = [i,
+                features[i].data.typimm+': '+features[i].data.nummai+' '+features[i].data.cadastre,
+                features[i].data.idemai,
+                features[i].data.source
+                ]
+            }
+
+            var store = new Ext.data.ArrayStore({
+                fields: ['index','displaytxt','idemai','idemai'],
+                data : properties 
+            });
+            
+            var combo = new Ext.form.ComboBox({
+                store: store,
+                displayField: 'displaytxt',
+                valueField: 'index',
+                typeAhead: true,
+                mode: 'local',
+                triggerAction: 'all',
+                emptyText: labels.choosePropertyMsg,
+                selectOnFocus:true,
+                listeners: {
+                    select: function(combo, record, index) {
+                        // the index of the selected feature is returned and used to highlight the geom
+                        property = features[index];
+                        Crdppf.featureSelection(property);
+                        Ext.getCmp('propertySelectionWindow').hide();
+                    }
+                }
+            });
+
+            if (Ext.getCmp('propertySelectionWindow')) {
```

Comment:
If you don't use `propertySelectionWindow` variable outside of this function, then you shouldn't use `Ext.getCmp`, but rather declare your variable locally inside the function and use the scope to hide it in the listeners.